docker-compose.yml

```diff
@@ -33,7 +33,7 @@ services:
       ## This shares the configuration with jobrunner
       - shared:/var/www/html/:rw
       - ./mediawiki/LocalSettings.d:/var/www/html/LocalSettings.d
-      - ./mediawiki//:/var/www/html/images/
```

Comment:
It would be better if this was a volume. Otherwise the files will be written to the host OS? This might cause a permission problem in the long run. Especially if we trasclude images from commons.

Reply:
as a network volume you mean ?